packages/core/src/State.ts

```diff
@@ -128,6 +131,7 @@ export class State<
    */
   public children: Record<string, ActorRef<any>>;
   public tags: Set<string>;
+  private machine: StateMachine<TContext, any, TEvent, any> | undefined;
```

Comment:
```suggestion
  private machine: StateMachine<TContext, any, TEvent, any> | undefined;
  public machine: StateMachine<TContext, any, TEvent, TTypestate> | undefined;
```

Reply:
@Andarist Deferring to you; would this be okay to do?